pepys_admin/admin_cli.py

```diff
@@ -25,7 +26,7 @@ def create_postgres_data_store():
         db_password="postgres",
         db_host="localhost",
         db_port=5432,

```

Comment:
could we keep the previous database name?

Reply:
Sure.